### PR TITLE
Added a feature that throws an error when a user enters an unsupported command.

### PR DESCRIPTION
**Pull request by lakshya8066**
Fixes (partially) #38

### CODE REVIEW

**Review by Atharva-Shinde — CHANGES_REQUESTED** [review]
Change the typo in error message - "argumant" to "argument"
 And add "an" before "input" in 'This command does not take any argumant as input.'

File: cmd/list.go [hunk 1]
```diff
@@ -29,6 +30,11 @@ var listCmd = &cobra.Command{
 	Long: `Gives a list of all the dependencies of the project. 
 	These include both direct as well as transitive dependencies.`,
 	RunE: func(cmd *cobra.Command, args []string) error {
+
+		if len(args) != 0 {
+			return errors.New("Invalid input! This command does not take any argumant as input.")
```
Comment (RinkiyaKeDad) [inline cmd/list.go]: I think we should go with `fmt.Errorf` because then we won't need to import another package here. Also please see #38 (comment)

**Review by RinkiyaKeDad — DISMISSED** [review]
Thanks for the PR! Just one minor nit and we should be good to go :)

File: cmd/list.go [hunk 2]
```diff
@@ -29,6 +29,11 @@ var listCmd = &cobra.Command{
 	Long: `Gives a list of all the dependencies of the project. 
 	These include both direct as well as transitive dependencies.`,
 	RunE: func(cmd *cobra.Command, args []string) error {
+
+		if len(args) != 0 {
+			return fmt.Errorf("list does not take any argument as input")
```
Comment (RinkiyaKeDad) [inline cmd/list.go]: `"list does not take any arguments"` for consistency 🙂

Comment (lakshya8066) [reply cmd/list.go]: oops, updating...

**Comment by RinkiyaKeDad** [comment]
Thanks!

/lgtm
/approve
/woof

**Pull request merged**